README.rst

```diff
@@ -1,35 +1,44 @@
-i18n Tools |build-status| |coverage-status|
-===========================================
+edX i18n Tools |build-status| |coverage-status|
+###############################################
 
 Installing
-----------
+==========
 
 ``python setup.py install``
 
 Running
--------
+=======
 
 Running commands from the edx-platform directory will default to loading the
 configuration at ``./conf/locale/config.yaml``. You can specify a different
 configuration file with the ``--config`` argument.
 
- * ``i18n_tool dummy``
- * ``i18n_tool extract``
- * ``i18n_tool generate``
- * ``i18n_tool segment``
- * ``i18n_tool transifex``
- * ``i18n_tool validate``
+* ``i18n_tool dummy``
+* ``i18n_tool extract``
+* ``i18n_tool generate``
+* ``i18n_tool segment``
+* ``i18n_tool transifex``
+* ``i18n_tool validate``
 
 
 Configuration
--------------
+=============
 
 Details of the config.yaml file are in `edx-platform/conf/locale/config.yaml
 <https://github.com/edx/edx-platform/blob/master/conf/locale/config.yaml>`_
 
 
+Changes
```

Comment:
Visually it is rendered as two peer sections, which is where I reviewed it:

![image](https://cloud.githubusercontent.com/assets/5985072/15079233/5afc86a6-1385-11e6-8564-9352319881af.png)